index.html

```diff
@@ -736,6 +736,7 @@ <h3>Role Mapping Table</h3>
 					<td class="role-axapi">
 						<span class="property">AXRole: <code>AXGroup</code></span><br />
 						<span class="property">AXSubrole: <code>AXDeleteStyleGroup</code></span><br />
+                                                <span class="property">AXAttributedStringForTextMarkerRange: contains <code>AXIsSuggestedDeletion = 1;</code> for all text contained in a <code>deletion</code></span>
```

Comment:
@cookiecrook I'd say the the terminology AXIsSuggestedDeletion/Insertion are a misnomer, because suggestions are only one use case. The insertion/deletion markup is useful for marking up a diff/history of a file. It's an insertion or deletion from the past.

Reply:
this is what webkit exposes, just to have the link on hand: https://github.com/WebKit/WebKit/blob/335a5587c31289a3a8043e4c460f7488c59e947b/Source/WebCore/accessibility/mac/WebAccessibilityObjectWrapperMac.mm#L855